interface/usergroup/mfa_registrations.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Multi-Factor Authentication Management
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2018 Rod Roark <rod@sunsetsystems.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE CNU General Public License 3
+ */
+
+require_once('../globals.php');
+
+use OpenEMR\Core\Header;
+
+function writeRow($method, $name)
+{
+    echo "        <tr><td>&nbsp;";
+    echo text($method);
+    echo "&nbsp;</td><td>&nbsp;";
+    echo text($name);
+    echo "&nbsp;</td><td>";
+    echo "<input type='button' onclick='delclick(\"" . attr($method) . "\", \"" .
+        attr($name) . "\")' value='" . xla('Delete') . "' />";
+    echo "</td></tr>\n";
+}
+
+$userid = $_SESSION['authId'];
+
+$message = '';
+if (!empty($_POST['form_delete_method'])) {
+    // Delete the indicated MFA instance.
+    sqlStatement(
+        "DELETE FROM login_mfa_registrations WHERE user_id = ? AND method = ? AND name = ?",
+        array($userid, $_POST['form_delete_method'], $_POST['form_delete_name'])
+    );
+    $message = xl('Delete successful.');
+}
+?>
+<html>
+<head>
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Manage Multi Factor Authentication'); ?></title>
+<script>
+
+function delclick(mfamethod, mfaname) {
+    var f = document.forms[0];
+    f.form_delete_method.value = mfamethod;
+    f.form_delete_name.value = mfaname;
+    top.restoreSession();
+    f.submit();
+}
+
+function addclick(sel) {
+    top.restoreSession();
+    if (sel.value) {
+        if (sel.value == 'U2F') {
+            window.location.href = 'mfa_u2f.php?action=reg1';
+        }
+        else {
+            alert('<?php echo xls('Not yet implemented.'); ?>');
+        }
+    }
+    sel.selectedIndex = 0;
+}
+
+</script>
+</head>
+<body class="body_top">
+<form method='post' action='mfa_registrations.php' onsubmit='return top.restoreSession()'>
```

Comment:
As promised, place the following code block here (please place it here since I am tracking these to be right below the form tag):
```
<input type="hidden" name="csrf_token_form" value="<?php echo attr(collectCsrfToken()); ?>" />
```